javaslang/src/main/java/javaslang/collection/AbstractMap.java

```diff
@@ -134,14 +134,7 @@ public M filter(Predicate<? super Tuple2<K, V>> predicate) {
     @SuppressWarnings("unchecked")
     @Override
     public <C> Map<C, M> groupBy(Function<? super Tuple2<K, V>, ? extends C> classifier) {
-        Objects.requireNonNull(classifier, "classifier is null");
```

Comment:
@danieldietrich, this part modifies production code (i.e. all the `groupBy` methods are unified and changed to keep insertion order.

It's needed for the benchmarks to display the final results in the original order.